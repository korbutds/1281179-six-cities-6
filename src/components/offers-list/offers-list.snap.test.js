import React from 'react';
import {render} from '@testing-library/react';
import {Test} from '../../test/test-mock';
import OffersList from './offers-list';
import {notEmptyStore} from '../../test/test-store';
test(`OffersList component render correctly`, () => {
  const {container} = render(
      <Test>
        <OffersList offers={notEmptyStore.CURRENT_OFFER.currentOffer.goods} />
      </Test>
  );
  expect(container).toMatchSnapshot();
});
