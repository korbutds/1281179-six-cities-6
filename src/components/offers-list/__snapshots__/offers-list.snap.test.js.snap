// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffersList component render correctly 1`] = `
<div>
  <ul
    class="property__inside-list"
  >
    <li
      class="property__inside-item"
    >
      Breakfast
    </li>
    <li
      class="property__inside-item"
    >
      Air conditioning
    </li>
    <li
      class="property__inside-item"
    >
      Laptop friendly workspace
    </li>
    <li
      class="property__inside-item"
    >
      Washer
    </li>
  </ul>
</div>
`;
